lms/djangoapps/courseware/module_render.py

```diff
@@ -755,6 +756,7 @@ def rebind_noauth_module_to_user(module, real_user):
             'milestones': milestones_helpers.get_service(),
```

Comment:
Is the `milestones` service still needed?

Reply:
I think we should leave it in, since any third-party XBlock is allowed to take advantage of it, and we won't have a full accounting of those.